# mayhem_program::__private::__global::unknown_2dcc  (yours 336 B at .text+0x73a30; original 456 B at .text+0x80dd0)
# distance 52 instructions; evidence: called-by mayhem_program::entry#c95e
--- original
+++ yours
@@ -1,56 +1,41 @@
 mov64 r7, r3
 mov64 r8, r2
 mov64 r6, r1
-lddw r1, str("Instruction: InitializeMayhemState")
-mov64 r2, 34
+lddw r1, str("Instruction: Unknown2dcc")
+mov64 r2, 24
 call sol_log_
-stw [r10-1540], -1
-stxdw [r10-1528], r7
-stxdw [r10-1536], r8
+stxdw [r10-584], r7
+stxdw [r10-592], r8
 mov64 r1, r10
-add64 r1, -1540
+add64 r1, -594
 stxdw [r10-4088], r1
 mov64 r1, r10
-add64 r1, -760
+add64 r1, -288
 mov64 r3, r10
-add64 r3, -1536
+add64 r3, -592
 mov64 r5, r10
 mov64 r2, r6
-call orig_fn_691d8
-ldxw r7, [r10-760]
-jne r7, 2, +0xc8
-ldxdw r1, [r10-744]
-ldxdw r0, [r10-752]
-ja +0x1c0
-ldxw r1, [r10-756]
-stxdw [r10-1552], r1
-ldxdw r9, [r10-752]
-ldxdw r8, [r10-744]
+call <mayhem_program::instructions::admin::Admin as anchor_lang::Accounts<mayhem_program::instructions::admin::AdminBumps>>::try_accounts#7802
+ldxdw r1, [r10-280]
+ldxdw r0, [r10-288]
+ldxb r7, [r10-3]
+jeq r7, 2, +0x148
+mov64 r3, r10
+add64 r3, -560
+mov64 r2, r10
+add64 r2, -272
+mov64 r8, r1
+mov64 r1, r3
+mov64 r3, 269
+mov64 r9, r0
+call memcpy
+stxb [r10-291], r7
+stxdw [r10-568], r8
+stxdw [r10-576], r9
+ldxh r1, [r10-2]
+stxh [r10-290], r1
 mov64 r1, r10
-add64 r1, -1496
-mov64 r2, r10
-add64 r2, -736
-mov64 r3, 736
-call memcpy
-stxdw [r10-1504], r8
-stxdw [r10-1512], r9
-ldxdw r1, [r10-1552]
-stxw [r10-1516], r1
-stxw [r10-1520], r7
-ldxw r1, [r10-1540]
-stxw [r10-728], r1
-ldxdw r1, [r10-1528]
-stxdw [r10-736], r1
-ldxdw r1, [r10-1536]
-stxdw [r10-744], r1
-mov64 r7, r10
-add64 r7, -1520
-stxdw [r10-752], r7
-stxdw [r10-760], r6
-mov64 r1, r10
-add64 r1, -760
-call entrypoint
-mov64 r1, r7
+add64 r1, -576
 mov64 r2, r6
-call orig_fn_6d4d0
+call <mayhem_program::instructions::admin::Admin as anchor_lang::AccountsExit>::exit#6c17
 exit
